styles.css

```diff
@@ -0,0 +1,194 @@
+@media only screen and (min-width: 375px) {
+  body {
+    width: 100%;
+  }
+
+  .appbar-compact {
+    position: absolute;
+    width: 375px;
+    height: 92px;
+    left: 0;
+    top: 0;
```

Comment:
- [ ] As the main container, I suggest t make this a flex. This gives you a lot of options to align all its Children elements, plus, it's responsive.

Reply:
👍